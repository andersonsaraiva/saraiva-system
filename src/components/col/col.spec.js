import { shallowMount } from '@vue/test-utils';
import Comp from './col.vue';

describe('col.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Comp);
    expect(wrapper.element).toMatchSnapshot();
  });

  it('valid props', () => {
    const wrapper = shallowMount(Comp, {
      props: {
        sm: 12,
        cols: 12
      }
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
