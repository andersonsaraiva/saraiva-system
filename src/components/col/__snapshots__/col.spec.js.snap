// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`col.vue matches snapshot 1`] = `
<div
  class="on-col"
>
  
  
</div>
`;

exports[`col.vue valid props 1`] = `
<div
  class="on-col-sm-12 on-col-12"
>
  
  
</div>
`;
